doc/source/reference/c-api/array.rst

```diff
@@ -1165,6 +1165,10 @@ User-defined data types
     *totype*. Any old casting function is over-written. A ``0`` is
     returned on success or a ``-1`` on failure.
 
+    .. c:type:: PyArray_VectorUnaryFunc
```

Comment:
Is the indentation correct?

Reply:
I avoided using the same indentation because the rest of this section is all function definitions.

Reply:
I think the [rendered indentation is correct](https://output.circle-artifacts.com/output/job/971c751b-d94a-4943-a791-32c2f46ad6d7/artifacts/0/doc/build/html/reference/c-api/array.html#c.PyArray_RegisterCastFunc.PyArray_VectorUnaryFunc). It seems like the [actual typedef of `PyArray_VectorUnaryFunc`](https://github.com/numpy/numpy/blob/441c9e600be16e1619ef6fb04e7ecdf6a12069cc/numpy/_core/include/numpy/ndarraytypes.h#L407) is not documented elsewhere so maybe this could be enhanced, but on the other hand it is surprisingly difficult to figure out what the signature means. Perhaps a link to [registering casting functions](https://output.circle-artifacts.com/output/job/971c751b-d94a-4943-a791-32c2f46ad6d7/artifacts/0/doc/build/html/user/c-info.beyond-basics.html#registering-a-casting-function) would be helpful on how to use this function.

Reply:
The function types should be documented somewhere, but I suppose the present location is a bit better than it was.